app/controllers/hostgroups_controller.rb

```diff
@@ -77,9 +76,10 @@ def destroy
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
+      hostgroup_parent_parameters = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_parent_id']).parameters(true) if params['hostgroup_parent_id']
+      hostgroup_parameters = params['hostgroup_id'] ? Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id']).group_parameters : []
       render :partial => "common_parameters/inherited_parameters",
-             :locals => { :inherited_parameters => inherited_parameters }
+             :locals  => { :inherited_parameters => hostgroup_parent_parameters, :parameters => hostgroup_parameters }
```

Comment:
since the comments disappeared, for the record this function was already called from all the right places on hostgroup but it didn't work (always returned 404 or 500 as @tbrisker mentioned). That is the reason for using it for the parent parameters.
I'm not against creating a new function and fixing this one but that makes sense only if hostgroup should show all of it's inherited parameters. Is there a bug about this? did it ever work? if not do we want to add this to hostgroup?